front/src/modules/activities/tasks/hooks/useCurrentUserDueTaskCount.ts

```diff
@@ -0,0 +1,47 @@
+import { DateTime } from 'luxon';
```

Comment:
@gitstart-twenty I'm actually introducing a new hook. Re-using tasks was a bad idea as useTasks is expecting filters to be set